src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -179,50 +179,48 @@
                DisplayName="5" />
   </EnumProperty>
 
+  <!-- TODO: Create fwlink for this HelpUrl -->
```

Comment:
@tmeschter created the last batch of fwlinks. We should create them in the same way, with owners configured so that we can easily update them in future.

Reply:
@MiYanni are you going to set up these fwlinks before merging this? If not, please file an issue so we don't forget to clean this up.

Reply:
HelpUrl changes were reverted.